src/store/initial_state.js

```diff
@@ -617,6 +624,28 @@ const state: GlobalState = {
                 error: null,
             },
         },
+        groups: {
```

Comment:
It looks like the default request state for `getAllChannels` and the 3 ldap/admin requests are missing.

Reply:
Good point.

Reply:
Fixed